README.md

```diff
@@ -261,6 +262,44 @@ module "api_gateway" {
 | redeployment_trigger                      | string       | A unique string to force a redeploy of the api gateway. If not set manually, the module will use the configurations for endpoints, api_key, and authorizer config to decide if a redeployment is necessary. | (yes)    |                                                           |
 | tags                                      | map(string)  | Map of tags to assign to the Lambda Function and related resources                                                                                                                                          | no       | `{}`                                                      |
 
+#### GitHub OpenID role
+
+```hcl
+module "github_action_role" {
+  source = "github.com/THEY-Consulting/they-terraform//aws/openid/github"
+
+  name = "they-test"
+  repo = "THEY-Consulting/they-terraform"
+  policies = [
+    {
+      name = "they-test-policy"
+      policy = jsonencode({
+        Version : "2012-10-17",
+        Statement : [
+          {
+            Effect : "Allow",
+            Action : [
+              "s3:ListBucket",
+            ],
+            Resource : [
+              "arn:aws:s3:::they-test-bucket",
+            ]
+          }
+        ]
+      })
+    },
+  ]
+}
+```
+
```

Comment:
QoL: Perhaps we could name this section `inputs` and add an `outputs` section, where applicable? Here I guess it would at least contain the role-arn.

Similar to what is done at e.g. aws module [here](https://registry.terraform.io/modules/terraform-aws-modules/iam/aws/latest/submodules/iam-assumable-role?tab=outputs) or for resources (with other naming) [here](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/appmesh_route#attribute-reference)